manifests/init.pp

```diff
@@ -22,6 +22,7 @@
   $dnskeyname                                             = undef,
   String $ddns_update_static                              = 'on',
   String $ddns_update_optimize                            = 'on',
+  String $ddns_client_updates                             = 'allow',
```

Comment:
I think the man page suggests this can be ```Enum['allow', 'deny']```. For the other reviewers: this default matches dhcpd.conf's default.